transcom/mymove
# B-18269 Added actual weight field

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/Default.aspx?menu=StoryBoardPage&feat-nav=--m2)
## [Agility Issue ticket](https://www13.v1host.com/USTRANSCOM38/Issue.mvc/Summary?oidToken=Issue%3A886897)

## Summary

This PR adds the prime actual weight field on the shipment weight screen.

### How to test

1. Create and Approve PPM Shipment
2. Might have to populate estimated weight and actual weight in the DB. Make sure there are values in those columns for the shipment.
3. As TIO review the weights on the shipment
4. You will notice actual weight under the estimated weight

![image](https://github.com/transcom/mymove/assets/136514590/38200ad3-9bc8-45a1-a4c9-0d58d41dd6f6)
